package.json

```diff
@@ -130,4 +130,4 @@
 			"@commitlint/config-conventional"
 		]
 	}
-}
+}
```

Comment:
This is just a formatting thing. I've copied the package.jsom from the master back to my branch as I've seen, that all this changes had been made in https://github.com/tidev/titanium/pull/558

Reply:
You're trimming trailing new lines. I prefer a empty line return at the end of the file.